### PR TITLE
add text and tooltip for copy success msg

### PR DESCRIPTION
**Pull request by sk-pathan**
fix:#1672

what pr do 
This pull request aims to enhance the user experience and streamline the development process by introducing two key features: "Try Deploy Locally" button and "Copy Tooltip for Success Message."

Changes Proposed:

Added "Try Deploy Locally" button to enable users to test deployments locally before committing changes.
Implemented a copy tooltip for success messages, providing users with clear feedback upon successful copy actions.

mobile view:
https://github.com/keploy/website/assets/141400463/c4f462ea-383a-4dc6-a47b-6adb55d56230

laptop view:

https://github.com/keploy/website/assets/141400463/5982f77b-d5f7-41fe-b83f-efc29b1772df

### CODE REVIEW

**Comment by sk-pathan** [comment]
hey @nehagup and @Hermione2408 review my changes

**Comment by Hermione2408** [comment]
Hey @sk-pathan 
 It is breaking in medium size please check 
Screenshot from 2024-03-31 14-30-29
Also please use "try keploy locally" color according to theme mentioned in README

**Comment by sk-pathan** [comment]
hey @Hermione2408 ,
I have fixed the issue and for colour of text i change the colour but you have better idea please let me know 
image

**Comment by Hermione2408** [comment]
Hey @sk-pathan  I noticed the second gif is overlapping in the medium size view. Also, there's inconsistency with the text color across different views; please make it consistent to text-secondary-300
Screenshot from 2024-04-01 17-04-55

**Comment by sk-pathan** [comment]
hi @Hermione2408 have look on change

File: components/features.tsx [hunk 1]
```diff
@@ -94,9 +107,23 @@ const CopyCode = () => {
 
   return (
     <>
+      <p className='font-bold text-secondry-300'>Try Keploy Locally </p>
```
Comment (Hermione2408) [inline components/features.tsx]: secondary*

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
Screenshot from 2024-04-03 11-43-25
Hey @sk-pathan In medium size, now gif is not aligned with the side div

File: components/features.tsx (hunk 1 above)
Comment (Hermione2408) [reply components/features.tsx]: @sk-pathan please correct this spelling mistake the class is not getting applied

**Review by Hermione2408 — CHANGES_REQUESTED** [review]
Hey due to fixed height to the image the gap in third gif is a lot in small screen can you make it more responsive? 
Screenshot from 2024-04-05 16-30-06

**Comment by sk-pathan** [comment]
@Hermione2408 
Black and White Groovy Keep Going Desktop Wallpaper

**Comment by Hermione2408** [comment]
Hey @sk-pathan, this issue is now closed https://github.com/keploy/keploy/issues/1672. Thanks for your contributions I am closing this PR!

**Pull request closed without merging**